app/views/formdata/UserFormData.java

```diff
@@ -0,0 +1,57 @@
+package views.formdata;
+
+import play.data.validation.ValidationError;
+import java.util.List;
+import java.util.ArrayList;
+
+/**
+ * Backing class for the User data form.
+ */
+public class UserFormData {
```

Comment:
LoginFormData is used for logging in and UserFormData is for creating accounts (so UserForm can have more fields like email). 
However, since these objects aren't really related to the models (we already have a User class in codeu.chat.common, I think what I will do is using DynamicForm to extract data instead `https://www.playframework.com/documentation/2.5.x/JavaForms`